src/beebop_deploy.py

```diff
@@ -176,9 +174,6 @@ def api_configure(container, cfg):
     else:
         print("[api] Downloading storage database")
         args = ["./scripts/download_db"]
-        if cfg.api_use_small_db:
-            args = args + ["--small"]
-        args = args + ["storage"]
```

Comment:
I think we can still include the storage line - though it looks like in the script, the storage location defaults to that anyway, we might as well keep this in to make the link with the volume clearer.

Reply:
ive removed this from beebop_py so the script doesn't accept arg